src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestRequest.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+/**
+ * The StreamingIngestRequest interface is a marker interface used for type safety in the {@link
+ * SnowflakeServiceClient} for streaming ingest API request.
+ */
+public interface StreamingIngestRequest {}
```

Comment:
Do all inheriting class have a request id field that can be moved into this interface?

Reply:
No, the configure request doesn't have request id.